cmd/v/help/shader.txt

```diff
@@ -0,0 +1,16 @@
+Usage:
+  v shader [options] [DIRECTORY / FILE]
+
+Examples:
+  v shader .
+  v shader examples/sokol/02_cubes_glsl/cube_glsl.v
```

Comment:
should it not be `cube_glsl.glsl` ?

Reply:
hm, both work 🤔

Reply:
Nope not as things work currently. I've done it like that to have the tool resemble `v <path or file>`. So vshader look up the files automatically

Reply:
I'm not sure if a shader file can be given actually 🤔

Reply:
I decided it to behave like that as a mild preparation in case it should be part of the main compile process